data/mods/Generic_Guns/magazines/gg_magazines_migration.json

```diff
@@ -193,7 +193,20 @@
     "replace": "pistol_speedloader"
   },
   {
-    "id": [ "a180mag", "mosquitomag", "ruger1022bigmag", "ruger1022mag", "sw22mag", "j22mag", "wp22mag", "kp32mag" ],
+    "id": [
+      "a180mag",
+      "a180mag1",
+      "a180mag2",
+      "a180mag3",
+      "a180mag4",
```

Comment:
you added your new mags to the gg_magazines_migration.json (migration files are where items go that will be phased out over time)

Reply:
No, this is correct. Generic guns is a mod that genericizes all guns by migrating existing guns and gun related items to generic versions.